lib/ledgerOperationStorage.js

```diff
@@ -35,6 +37,57 @@ module.exports = class LedgerOperationStorage {
       });
   }
 
+  // TODO: an optional parameter like
+  // `since: {blockHeight: 0, blockOrder: 0, eventOrder: 0}`
+  // could be useful if the state machine only needs operations after a certain
+  // point
```

Comment:
Along these lines^ ... we have an immediate need for getting a record "at blockHeight x".